### DISCUSSION TITLE
How should debugging / logging be setup for Prisma?

### DISCUSSION THREAD

**Discussion by barrybtw** [CodingGarden/listd#51] [Q&A]
Currently we have _no_ logging setup, what do you think would be a good idea, should it be opt-in or opt-out, and how should users be able to configure log settings?

**Comment by brilvio** [comment]
In my work we have a env var `LOG_LEVEL` where you can pass info, warning, error, debug, with this we can set the log level for the project as whole not just for Prisma

(edit: added debug to the list)
